### ISSUE TITLE
Add support for IIFEs

### ISSUE THREAD

**Issue by jestrux** [yatki/vscode-surround#50]
Thank you for the plugin, 

I was recently working on an old AngularJS project and a lot of the code wasn't IIFE-ed....I wrote a custom wrapper for IIFEs and I thought since it's a popular javascript pattern, you might want to add it to the main wrappers.

Here is my implementation:
```json
"IIFE": {
    "label": "IIFE",
    "description": "Wrap with Immediately Invoked Function",
    "snippet": "(function(){\n\n\t$TM_SELECTED_TEXT\n\n})();$0"
}
```

**Comment by yatki** [comment]
@jestrux Thank you very much for the suggestion. Loved it. I'll include it to the next release.

**Comment by jestrux** [comment]
Hey, you're fast, hadn't even closed this page yet. Glad you liked the suggestion, really easy to use and customize plugin you have here.

**Comment by Tiencrypt** [comment]
tovantien15121995@gmail.com

**Comment by yatki** [comment]
🎉 This issue was resolved in version 1.1.1 🎉

The release is available on: 
- [VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=yatki.vscode-surround)
- [GitHub release](https://github.com/yatki/vscode-surround/releases/tag/v1.1.1)

Don't forget to update your extension and leave a star :)
Cheers🖖🏻,
